src/main/java/core/basesyntax/service/serviceimpl/DataConverterImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    public static final String SEPARATOR = ",";
+    public static final int REPORT_START_POSITION_INDEX = 1;
+    public static final int OPERATION_INDEX = 0;
+    public static final int FRUIT_NAME_INDEX = 1;
+    public static final int QUANTITY_INDEX = 2;
+
+    @Override
```

Comment:
The method implementation should not begin with an empty line. Please remove the empty line at the beginning of the method body.